content/en/docs/concepts/services-networking/_index.md

```diff
@@ -54,6 +54,8 @@ Kubernetes networking addresses four concerns:
   to be reachable from outside your cluster.
   - [Ingress](/docs/concepts/services-networking/ingress/) provides extra functionality
     specifically for exposing HTTP applications, websites and APIs.
+  - [Gateway](/docs/concepts/services-networking/gateway/) provides an expressive, extensible,
+    and role-oriented API for modeling service networking.
```

Comment:
I would mention that Gateway is an extension. People won't otherwise guess that a vanilla cluster doesn't include the APIs from Gateway API.

Reply:
I am using the term `add-on` instead of extension since Gateway API uses the term extension.